src/core/unittest/RecvBufferTest.cpp

```diff
@@ -560,8 +563,14 @@ TEST_P(WithMode, ReadPendingMultiWrite)
     RecvBuf.Read(&ReadOffset, &BufferCount, ReadBuffers);
     ASSERT_FALSE(RecvBuf.HasUnreadData());
     ASSERT_EQ(32ull, ReadOffset);
-    ASSERT_EQ(1u, BufferCount);
-    ASSERT_EQ(48u, ReadBuffers[0].Length);
+    if (Mode == QUIC_RECV_BUF_MODE_MULTIPLE) {
```

Comment:
Why is this needed?

Reply:
Default AllocLength is 64. Then

Write 0,32
Read
Write 32,48 (append next chunk)
Drain 32
Read -> 32 from first chunk, 16 from second chunk.